server/src/models/form.model.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+
+const formSchema = new mongoose.Schema({
+  name: String,
+  email: String,
+  phone: String,
+  address: String,
+  timeRange: {
+    earlyTimeHour: Number,
+    lateTimeHour: Number,
+  },
+  dateCreated: { type: Date, default: Date.now },
+  status: {
+    type: String,
+    enum: ["Pending", "Confirmed", "Cancelled", "Visited"],
+    default: "Pending",
+  },
+});
+
+const Form = mongoose.model("Form", formSchema, "forms");
```

Comment:
I'd suggest this schema/concept should probably be called Appointment. Or I thought maybe AppointmentForm, but I don't think the "form" name makes sense when Visited is one of the options